doc/README.isql_enhancements.txt

```diff
@@ -353,3 +353,69 @@ CON>     select id from employees where id = ? into id;
 CON> end!
 SQL>
 SQL> set term ;!
+
+
+13) SET AUTOTERM ON/OFF
+
+Author: Adriano dos Santos Fernandes
+
+When set to ON, terminator defined with SET TERM is changed to semicolon and a new logic
+for TERM detection is used, where engine helps ISQL to detect valid usage of semicolons
+inside statements.
+
+At each semicolon (outside quotes or comments), ISQL prepares the query buffer with
+engine using flag IStatement::PREPARE_REQUIRE_SEMICOLON.
+
+If engine prepares the statement correctly, it's run and ISQL is put in new statement
+mode.
+
+If engine returns error isc_command_end_err2, then ISQL is put in statement
+continuation mode and asks for another line, repeating the process.
+
+If engine returns a different error, the error is shown and ISQL is put in new statement
+mode.
+
+Notes:
+- This option can also be activated with command line parameter -autot(erm)
+- It can only be used with Firebird engine/server v6 or later
+- SET TERM command automatically sets AUTOTERM to OFF
+- SET AUTOTERM ON command automatically sets TERM to semicolon
```

Comment:
What terminator will be set after SET AUTOTERM OFF ?
Will it remains semicolon or will be returned to the state as before SET AUTOTERM ON ?

Reply:
I put in the docs that it is set to semicolon. For me it was clear that it's not reverted when AUTOTERM is turned OFF, but we may improve it.

Reply:
Yes, it was about a bit more clear docs.